modules/emulator/R/minimize.GP.R

```diff
@@ -97,6 +97,9 @@ gpeval <- function(xnew, k, mu, tau, psi, x, rng, splinefcns) {
 ##' @name ddist
 ##' @title ddist
 ##' @export
+##' 
+##' @param x Name of variable to plot on X axis
```

Comment:
vector of values (e.g. observations) to be evaluated by the specified probability density function